src/bitfield/api.ts

```diff
@@ -0,0 +1,22 @@
+export interface BitField<Self extends unknown = unknown> {
```

Comment:
There is a neighbor to this file called `api.js` that is empty. Should that empty file be removed?

Reply:
No it's just a trick to `import * as API from "./api.js"` and get all the types under `API` namespace.